src/App.tsx

```diff
@@ -15,69 +15,141 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type State = {
+  isReversed: boolean;
+  sortType: SortType;
+};
+
 enum SortType {
   NONE,
   ALPHABET,
   LENGTH,
 }
 
-type ReorderOptions = {
-  sortType: SortType;
-  isReversed: boolean;
-};
-
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
```

Comment:
Let's use switch statement here, as sorting options may grow - so will your if-else structure do. 
switch is just a more generic way to approach it.